### PR TITLE
Azure workload Identity - private lb

### PR DESCRIPTION
**Pull request by huangmingxia**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 1]
```diff
@@ -385,6 +385,42 @@ tests:
...
+      E2E_RUN_TAGS: '@amd64 and @azure-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips'
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      TAG_VERSION: '@4.14'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+    test:
+    - ref: ipi-install-heterogeneous
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate-cco-manual-workload-identity
+- as: azure-ipi-disc-fullypriv-oidc-arm-f14-destructive
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:405]: f28?

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 2]
```diff
@@ -385,6 +385,42 @@ tests:
...
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @azure-ipi and @disconnected and @network-ovnkubernetes
+        and not @fips'
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      TAG_VERSION: '@4.14'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&
+    test:
+    - ref: ipi-install-heterogeneous
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:402]: I do not think we need this step.

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disconnected/fullyprivate/cco-manual-workload-identity/deprovision/cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate-cco-manual-workload-identity-deprovision-chain.metadata.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+	"path": "cucushift/installer/rehearse/azure/ipi/disconnected/fullyprivate/cco-manual-workload-identity/deprovision/cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate-cco-manual-workload-identity-deprovision-chain.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/disconnected/fullyprivate/cco-manual-workload-identity/deprovision/cucushift-installer-rehearse-azure-ipi-disconnected-fullyprivate-cco-manual-workload-identity-deprovision-chain.metadata.json:2]: I know we recently we hit some issues on azure + oidc + private cluster, so I wonder can we reproduce the issue without `disconnected`?

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-priv-oidc-amd-f14

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-multi-nightly-azure-ipi-disc-fullypriv-oidc-amd-f14

**Pull request closed without merging**